### ISSUE TITLE
trouble while installing MoCha

### ISSUE THREAD

**Issue by zedicush** [freeseek/mocha#50]
Hello,

I'm getting trouble while installing MoCha with this command line. 
" cd bcftools-1.20/
/bin/rm -f plugins/{{mocha,beta_binom,genome_rules}.h,{mocha,mochatools,extendFMT}.c}
wget -P plugins https://raw.githubusercontent.com/freeseek/mocha/master/{{mocha,beta_binom,genome_rules}.h,{mocha,mochatools,extendFMT}.c}
make
/bin/cp bcftools plugins/{fill-tags,fixploidy,mocha,trio-phase,mochatools,extendFMT}.so $HOME/bin/ "

The error is /bin/cp bcftools plugins/{fill-tags,fixploidy,mocha,trio-phase,mochatools,extendFMT}.so $HOME/bin/
/bin/cp: cannot stat 'plugins/trio-phase.so': No such file or directory

After the make, I don't see the trio-phase in the plugins.
thank you

**Comment by freeseek** [comment]
Yeah sorry, the trio-phase plugin has been sunset as the same capability is now available in SHAPEIT5. Just remove it from the command line. I will update the documentation

**Comment by zedicush** [comment]
Thank you for the quick answer
